macros/redshift_maintenance_operation.sql

```diff
@@ -0,0 +1,27 @@
+{% macro get_vacuumable_tables() %}
```

Comment:
We could parameterize this further, say, have a schema condition, or a max column width condition.

I am not too worried about doing this in the package though -- anyone who is doing this sort of work should feel comfortable about how to add a macro to their own project.\